templates/velero.yaml

```
@@ -7,391 +28,47 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: velero
 spec:
+  namespace: velero
   kubernetes:
     minSupportedVersion: v1.14.0
   requires:
     matchLabels:
       kubeaddons.mesosphere.io/provides: defaultstorageclass
```

Since the values are all aws specific, let's add this addon to the configs of the other cloud providers, but leave it disabled:

```suggestion
      kubeaddons.mesosphere.io/provides: defaultstorageclass
    cloudProvider:
    - name: aws
      enabled: true
    - name: docker
      enabled: false
    - name: none
      enabled: false
```

---

we add default values to other charts in kubeaddons (like metallb). is it possible to do that for this chart?

---

and I don't know if that suggestion thing is working. I show six spaces before `kubeaddons` before the suggestion and after.